### DISCUSSION TITLE
extremely long log file entries created - need ability to suppress $stdout

### DISCUSSION THREAD

**Discussion by douginoz** [meisnate12/Plex-Meta-Manager#274] [Ideas]
I've got pmm running at bootup on ubuntu 20.04.  The system log file /var/log/syslog gets insanely huge entries generated every few minutes from pmm because it's trying to write out it's current countdown status.  I've got PMM configured to run at 3am every day. I won't paste in the entirety of one entry, but here's the start of an example.  Note that the actual one is about 50x longer:
`
May 24 14:25:10 sophie python3[1630]: rs and 47 Minutes until the daily run at 03:00#015| Current Time: 14:13 | 12 Hours and 47 Minutes until the daily run at 03:00#015| Current Time: 14:13 | 12 Hours and 47 Minutes until the daily run at 03:00#015| Current Time: 14:13 | 12 Hours and 47 Minutes until the daily run at 03:00#015| Current Time: 14:13 | 12 Hours and 47 Minutes until the daily run at 03:00#015| Current Time: 14:13 | 12 Hours and 47 Minutes until the daily run at 03:00#015| Current Time: 14:13 | 12 Hours and 47 Minutes until the daily run at 03:00#015| Current Time: 14:13 | 12 Hours and 47 Minutes until the daily run at 03:00#015| Current Time: 14:13 | etc etc`

The plexmetamanager.service file is as basic as possible and the execStart line is 
`ExecStart=/usr/bin/python3 /opt/Plex-Meta-Manager/plex_meta_manager.py`

I think what it needs is a runtime option to tell it not to display that ~48K of countdown timer info.

**Comment by meisnate12** [comment]
i didnt realize it would create such big logs like that I can help out it'll be in my next release

**Reply by meisnate12** [reply]
ok i added `--no-countdown` or `-nc` as a run argument which will not display the countdown
